setup.py

```diff
@@ -8,6 +8,12 @@
     long_description = f.read()
 with open(os.path.join(path, "requirements-minimal.txt")) as f:
     install_requires = f.read().strip().split("\n")
+with open(os.path.join(path, "requirements-full.txt")) as f:
+    test_requires = f.read().strip().split("\n")
+test_requires += ["pytest", "pytest-cov"]
```

Comment:
why not have these in requirements-full.txt?

Reply:
I am not really sure. I wanted to do that but I was afraid it would break some of the CI stuff. I am hoping that can @CodyCBakerPhD comment on it.

Reply:
`requirements-full.txt` is the list of legit external dependencies a user might need for utilizing the `DataInterfaces`. `pytest` is only needed for developers looking to run the full testing suite. `pytest-cov` is only for the CI to compile reports for upload.

So I'd prefer `pytest` and `pytest-cov` to be developer-specific installations outside requirement files. Or a separate `requirements-dev.txt` but that seems excessive.